codegen/core/src/main/java/software/amazon/smithy/python/codegen/writer/MarkdownToRstDocConverter.java

```diff
@@ -125,41 +151,47 @@ public void tail(Node node, int depth) {
                 Element element = (Element) node;
                 switch (element.tagName()) {
                     case "a":
-                        sb.append(" <").append(element.attr("href")).append(">`_");
+                        String href = element.attr("href");
+                        if (!href.isEmpty()) {
+                            writer.writeInline(" <").writeInline(href).writeInline(">`_");
+                        } else {
+                            writer.writeInline("`");
+                        }
                         break;
                     case "b":
                     case "strong":
-                        sb.append("**");
+                        writer.writeInline("**");
                         break;
                     case "i":
                     case "em":
-                        sb.append("*");
+                        writer.writeInline("*");
                         break;
                     case "code":
```

Comment:
We should also add a check for code blocks (`<pre><code>`). A handful of services (including bedrock-runtime) use these, though sadly there seems to be no syntax indicators.